addons/fleet/models/res_partner.py

```diff
@@ -7,4 +7,4 @@
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
-    plan_to_change_car = fields.Boolean('Plan To Change Car', default=False)
+    plan_to_change_vehicle = fields.Boolean('Plan To Change Vehicle', default=False)
```

Comment:
I don't like the idea of mixing the intention to change car and bike. 
- What happen if you want to change both ?
- How do you recognize if it's the car or the bike that should change ? 

On the fleet.vehicle, this is a related field, so now that we assume we could drive several different vehicles at the same time, planning to change your bike will transfer this information to the company car too.
And with this line: https://github.com/odoo/enterprise/blame/master/l10n_be_hr_payroll_fleet/models/hr_contract.py#L29
It also means that if you plan to change your bike, you'll reintroduce your car on the salary configurator :)

Or maybe I'm missing something ?